src/frontend/babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    "presets": [
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
+                    "node": 18
+                }
+            }
+        ]
+    ]
+}
```

Comment:
```suggestion
}

```

Reply:
Thanks Brandon! Corrected the same with d3c9041e260b1359ac12f5d9aecc2ed0bb515d0e